projects/output-and-error-handling/client/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net/http"
+)
+
+func main() {
+	res, err := http.Get("http://localhost:8080")
+	if err != nil {
+
+		if res != nil {
+			fmt.Println("Response status:", res.Status)
+			if res.StatusCode == http.StatusTooManyRequests {
+				fmt.Println("Server is too busy. Retry after:", res.Header.Get("Retry-After"))
+			} else {
+				fmt.Println("Failed to get response from server:", err)
```

Comment:
Should this kind of logging be printed to stdout or stderr? Which is it getting printed to here?